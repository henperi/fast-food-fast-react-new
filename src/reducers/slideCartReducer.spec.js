/* eslint-disable no-undef */
import slideCartReducers, { initialState } from './slideCartReducer';

describe('slideCartReducers', () => {
  it('should return the default state if no action is specified', () => {
    const action = { type: '' };
    expect(slideCartReducers(initialState, action)).toMatchSnapshot({});
    expect(slideCartReducers(initialState, action)).toEqual(initialState);
  });

  it('should set the slideCart state to true when type is SLIDE_CART and payload is specified', () => {
    const action = {
      type: 'SLIDE_CART',
    };
    expect(slideCartReducers(initialState, action)).toMatchSnapshot({});
    expect(slideCartReducers(initialState, action)).toEqual({ slide: true });
  });
});
