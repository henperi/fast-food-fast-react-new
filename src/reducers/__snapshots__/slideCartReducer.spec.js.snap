// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`slideCartReducers should return the default state if no action is specified 1`] = `
Object {
  "slide": false,
}
`;

exports[`slideCartReducers should set the slideCart state to true when type is SLIDE_CART and payload is specified 1`] = `
Object {
  "slide": true,
}
`;
